Engineering/SourceControlDetails.md

```diff
@@ -120,7 +120,7 @@ Let's use the following naming conventions for branches:
  * release branches: `release/release_name`
 
 ## Restrict access to default branches
-* In VSTS it's recommended to specify the **minimum number of required reviewers** as well as the **commit strategy** if you want to enforce [squash merging](https://docs.microsoft.com/en-us/vsts/git/merging-with-squash?view=vsts). Here some details on [VSTS branch policies](https://docs.microsoft.com/en-us/vsts/git/branch-policies?view=vsts)
+* In Azure DevOps it's recommended to specify the **minimum number of required reviewers** as well as the **commit strategy** if you want to enforce [squash merging](https://docs.microsoft.com/en-us/vsts/git/merging-with-squash?view=vsts). Here some details on [Azure DevOps branch policies](https://docs.microsoft.com/en-us/vsts/git/branch-policies?view=vsts)
```

Comment:
nit: should update link to `https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops`

Reply:
Link updated